client/components/mma/accountoverview/PersonalisedHeader.tsx

```diff
@@ -50,6 +50,7 @@ export const PersonalisedHeader = ({
 					margin-bottom: 0;
 				`}
 			>
+				{/*add in Quantum Metrics masking tag when name is piped through*/}
```

Comment:
still todo?

Reply:
I think it makes sense to add the masking now as there's the possibility that the supporter name is shown if it exists in the API response.